tests/test_redir.py

```diff
@@ -283,47 +284,51 @@ def test_offline_compression(test_settings, capfd, runner, console):
 
 @pytest.mark.parametrize("console", console_modes)
 @pytest.mark.parametrize("numpy", [True, False])
-@pytest.mark.timeout(120)
+@pytest.mark.timeout(300)
```

Comment:
why do you need a longer timeout?

Reply:
It failed for me locally once, so I ++'ed it

Reply:
ok if you think it makes sense :)